src/item_factory.cpp

```diff
@@ -931,7 +929,7 @@ void Item_factory::load_basic_info(JsonObject &jo, itype *new_item_template)
     if (jo.has_member("name_plural")) {
         new_item_template->name_plural = jo.get_string("name_plural").c_str();
     } else {
-        // default behaviour: Assume the regular plural form (appending an “s”)
+        // default behaviour: Assume the regular plural form (appending an �s�)
```

Comment:
Your editor destroyed the perfectly valid Unicode characters (left and right double quotation mark). Whatever it is now, it does not appear to be valid UTF-8. Please instruct your editor to open and save the source files as UTF-8 (there should be a setting named encoding or similar).